tests/x11regressions/firefox/firefox_java.pm

```diff
@@ -16,45 +16,44 @@ use base "x11regressiontest";
 use testapi;
 
 sub java_testing {
+    my ($self) = @_;
+
     send_key "ctrl-t";
     send_key "alt-d";
     type_string "http://www.java.com/en/download/installed.jsp?detect=jre\n";
 
     wait_still_screen 3;
+    check_screen('oracle-cookies-handling');
+    if (match_has_tag 'oracle-cookies-handling') {
+        assert_and_click('firefox-java-agree-and-proceed');
+    }
```

Comment:
Rather `assert_and_click 'firefox-java-agree-and-proceed' if check_screen 'oracle-cookies-handling', 0;` if the `wait_still_screen 3` above is enough. But a safer approach should be the multi-tag assert_screen with match_has_tag